src/WebGPURenderer.ts

```diff
@@ -11,27 +26,51 @@ export default function WebGPURenderer(loader: unknown) {
   this._tempb = new Bounds();
 }
 
-const base = Renderer.prototype;
+let base = Renderer.prototype;
```

Comment:
This reminds me to convert Renderer to a class rather than just taking the prototype.